R/ui-legacy.R

```diff
@@ -291,7 +289,7 @@ indent <- function(x, first = "  ", indent = first) {
   paste0(first, x)
 }
 
-ui_bullet <- function(x, bullet = cli::symbol$bullet) {
+ui_legacy_bullet <- function(x, bullet = cli::symbol$bullet) {
```

Comment:
I can't delete this function because it powers the legacy `ui_*()` functions. But I can rename it so it doesn't compete with `ui_bullets()` in auto-completion, which makes working on usethis more pleasant.